src/PolyhedralGeometry/PolyhedralComplex/standard_constructions.jl

```diff
@@ -75,3 +75,68 @@ function k_skeleton(PC::PolyhedralComplex{T}, k::Int) where {T<:scalar_types}
   )
   return PolyhedralComplex{T}(ksk, coefficient_field(PC))
 end
+
+
+###############################################################################
+## Scalar product
+###############################################################################
+
+function *(c::QQFieldElem, Sigma::PolyhedralComplex)
+    # if scalar is zero, return polyhedral complex consisting only of the origin
+    if iszero(c)
+        return polyhedral_complex(convex_hull(zero_matrix(QQ,1,ambient_dim(Sigma))))
+    end
+
+    # if scalar is non-zero, multiple all vertices and rays by said scalar
+    SigmaVertsAndRays = vertices_and_rays(Sigma)
+    SigmaRayIndices = findall(vr -> vr isa RayVector, SigmaVertsAndRays)
+    SigmaLineality = lineality_space(Sigma)
+    SigmaIncidence = maximal_polyhedra(IncidenceMatrix,Sigma)
+    return polyhedral_complex(SigmaIncidence, multiply_by_nonzero_scalar.(SigmaVertsAndRays,c), SigmaRayIndices, SigmaLineality)
+end
+*(c::Union{ZZRingElem,Rational,Int}, Sigma::PolyhedralComplex) = QQ(c)*Sigma
+*(Sigma::PolyhedralComplex, c::Union{QQFieldElem,ZZRingElem,Rational,Int}) = c*Sigma
```

Comment:
```suggestion
*(c::RationalUnion, Sigma::PolyhedralComplex) = QQ(c)*Sigma
*(Sigma::PolyhedralComplex, c::RationalUnion) = c*Sigma
```
this union already has a name